manticore/seth.py

```diff
@@ -1,97 +1,52 @@
-from manticore import Manticore
-from manticore.core.smtlib import ConstraintSet, Operators, solver, issymbolic, Array, Expression, Constant
-from manticore.core.smtlib.visitors import arithmetic_simplifier
-from manticore.platforms import evm
-from manticore.core.state import State
+from . import Manticore
+from .core.smtlib import ConstraintSet, Operators, solver, issymbolic, Array, Expression, Constant
+from .core.smtlib.visitors import arithmetic_simplifier
+from .platforms import evm
+from .core.state import State
 import tempfile
 from subprocess import Popen, PIPE
 import sha3
 import json
+import StringIO
 
 
-class EVMContract(object):
+class ABI(object):
```

Comment:
this should be a module, not a class -let's fix in a later PR though

Reply:
If we go for a multi file solution then yes.